web/script.js

```diff
@@ -48,18 +60,20 @@
         for(var k = 0; k < res.length; k++) {
           var measurement = res[k];
           var el = document.createElement("div");
+          var link = getLogLink(result.servers[j], result.clients[i], measurement.name, measurement.abbr);
           switch(measurement.result) {
             case "succeeded":
               el.className = "text-success";
-              el.innerHTML = measurement.name + ": " + measurement.details;
+              el.appendChild(link);
```

Comment:
Does this link to the letters themselves?

Reply:
Yes. It's already running on the website.